typescript-forms/form-values.ts

```diff
@@ -0,0 +1,25 @@
+interface HTMLFormControlsCollection {
```

Comment:
- HTMLFormControlsCollection type already exists. When you access the .elements property on an HTML form element, the type of it is assumed to be of HTMLFormControlsCollection. So you want to remove this interface definition lines 1-5... You define your own interface FormElements as an extension of  HTMLFormControlsCollection type to say it's will specifically contain properties name, email, and message...